pkgs/by-name/in/infisicalsdk/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "infisicalsdk";
+  version = "1.0.8";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Infisical";
+    repo = "python-sdk-official";
+    tag = "v${version}";
+    hash = "sha256-C8pHypKVPISDGGUlC2LXP4MPoae/ZS7Mb92yN7+VoPM=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
+  dependencies = with python3Packages; [
+    python-dateutil
+    aenum
+    requests
+    boto3
+    botocore
+  ];
+
```

Comment:
can you add `pythonImportsCheck = [ "infisical_sdk" ];`? Also there doesn't seems to be any tests, so `doCheck = false;`